samples/CMakeLists.txt

```diff
@@ -70,6 +74,10 @@ if(WIN32)
   add_subdirectory(directx)
 endif()
 
+if(NOT ANDROID)
+  add_subdirectory(opengl)
+endif()
```

Comment:
@avershov, try to remove this block (lines 77-80), because `opengl` directory is already added on lines 21-24. It should fix standalone samples build.